instructor/common.py

```diff
@@ -1097,6 +1100,88 @@ def facs_sample_prep(request):
         context_instance=RequestContext(request))
 
 
+def create_facs_histograms(assignment, facs_sample):
```

Comment:
can you please add a docstring describing what this function does (with more details than just the function name)?